### PR TITLE
Mailhog support

### PR DESCRIPTION
**Pull request by danquah**

### CODE REVIEW

File: files/etc/my_init.d/setup-mail.sh [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+# Ensure Blackfire is only enabled if we have the necessary configuration.
```
Comment (xendk) [inline files/etc/my_init.d/setup-mail.sh]: Wha?

File: files/etc/php/7.0/mods-available/mailpath.ini [hunk 2]
```diff
@@ -0,0 +1,2 @@
+; priority=90
+sendmail_path = ${PHP_SENDMAIL_PATH}
```
Comment (xendk) [inline files/etc/php/7.0/mods-available/mailpath.ini:2]: Variables work here?

**Review by xendk — APPROVED** [review]
Odd comment...

File: files/etc/php/7.0/mods-available/mailpath.ini (hunk 2 above)
Comment (danquah) [reply files/etc/php/7.0/mods-available/mailpath.ini:2]: Yes :)

File: files/etc/my_init.d/setup-mail.sh (hunk 1 above)
Comment (danquah) [reply files/etc/my_init.d/setup-mail.sh]: Sorry about that :) fixed.

File: files/etc/php/7.0/mods-available/mailpath.ini (hunk 2 above)
Comment (danquah) [reply files/etc/php/7.0/mods-available/mailpath.ini:2]: (see http://php.net/manual/en/configuration.file.php#example-36)

**Review by xendk — APPROVED** [review]
:+1:

**Pull request merged**